io.github.ryanabx.flatrun.yml

```diff
@@ -0,0 +1,70 @@
+id: io.github.ryanabx.flatrun
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+base: org.flatpak.Builder.BaseApp
+base-version: "23.08"
+separate-locales: false
+command: flatrun
+finish-args:
+  - "--share=ipc"
+  - "--device=dri"
+  - "--socket=wayland"
+  - "--socket=fallback-x11"
+# Unfortunately, it's necessary for us to have this permission
+# all unsandboxed access is run through flatrun-agent.
+# flatrun-agent installs system runtimes and launches flatpaks
+# from a local repo (non system and non user).
+  - "--talk-name=org.freedesktop.Flatpak"
+build-options:
+  append-path: "/usr/lib/sdk/rust-stable/bin"
+  env:
+    CARGO_HOME: "/run/build/flatrun/cargo"
+modules:
+# Install flatpak
+  - name: flatpak
+    config-opts:
+      - "--disable-documentation"
+      - "--disable-seccomp"
+      - "--disable-sandboxed-triggers"
+      - "--disable-system-helper"
+      - "--with-system-install-dir=/var/lib/flatpak"
+    cleanup:
+      - "/etc/profile.d"
+      - "/lib/systemd"
+      - "/share/dbus-1/interfaces/org.freedesktop.*"
+      - "/share/dbus-1/services/org.freedesktop.*"
+      - "/share/gdm"
+    sources:
+      - type: archive
+        url: "https://github.com/flatpak/flatpak/releases/download/1.14.8/flatpak-1.14.8.tar.xz"
+        sha256: "1016b7327f7af87896f95465f7e5813750d3b7049a3740a1a4ddcb5fa8c5348e"
+        x-checker-data:
+          type: "json"
+          url: "https://api.github.com/repos/flatpak/flatpak/releases/latest"
+          version-query: ".tag_name"
+          url-query: "\"https://github.com/flatpak/flatpak/releases/download/\\($version)/flatpak-\\($version).tar.xz\""
+# Install flatrun
+  - name: flatrun
+    buildsystem: simple
+    build-commands:
+      # Build using BaseApp libs and binaries
+      - cargo --offline fetch --manifest-path Cargo.toml --verbose
+      - cargo --offline build --release --verbose
+      # Cleanup BaseApp as we don't need it any more
+      - rm -rf /app/*
+      # Install binary and data
+      - install -Dm0755 target/release/flatrun /app/bin/flatrun
+      - install -Dm0755 target/release/flatrun-agent /app/libexec/flatrun-agent
+      - install -Dm0644 data/flatrun.desktop /app/share/applications/io.github.ryanabx.flatrun.desktop
+      - install -Dm0644 data/flatrun-bundle.desktop /app/share/applications/io.github.ryanabx.flatrun-bundle.desktop
+      - install -Dm0644 data/flatrun.metainfo.xml /app/share/metainfo/io.github.ryanabx.flatrun.metainfo.xml
```

Comment:
remove `xml:lang=en` from screenshots, see the note here https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#metainfo-translations

and the url is missing v in the tag

Reply:
Done! Thanks for pointing that out! https://github.com/ryanabx/flatrun/commit/eb5f9f26910030f747d22ec7a7bbd94c343cdfd4

Reply:
Thought I mentioned it already, but you are missing OARS too https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#oars-information

Reply:
Fixed and retagged the release: https://github.com/ryanabx/flatrun/commit/e36d25d419413584c659cb5360f887e81ea0324c

Reply:
retagging is bad.